src/EditorFeatures/Core/Implementation/Suggestions/SuppressionSuggestedAction.cs

```diff
@@ -119,7 +119,7 @@ public string GetDiagnosticID()
             }
 
             // if it is from third party, we use hashcode
-            return diagnostic.GetHashCode().ToString(CultureInfo.InvariantCulture);
+            return diagnostic.Id.GetHashCode().ToString(CultureInfo.InvariantCulture);
```

Comment:
diagnostic.GetHashCode() contains location which can change as you change unrelated code. which make telemetry data pretty much useless.